proxy/ui.coffee

```diff
@@ -69,6 +69,16 @@ class WebExtUI extends UI
       @postActive()
     ), 60 * 60 * 1000
 
+  initToggle: ->
+    getting = chrome.storage.local.get("snowflake-enabled", (result) =>
+      if result['snowflake-enabled'] != undefined
+        @enabled = result['snowflake-enabled']
+      else
+        @enabled = true
```

Comment:
You can remove his line since `@enabled` is already `true` from L7 and it's probably best to just let be whatever the default is.

Reply:
Thanks, fixed :)